eZ/Publish/API/Repository/Tests/SearchService/SearchServiceFullTextEmbedTest.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Tests\SearchService;
+
+use eZ\Publish\API\Repository\SearchService;
+use eZ\Publish\API\Repository\Tests\BaseTest;
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\Core\FieldType\RichText\Value as RichTextValue;
+use eZ\Publish\Core\Repository\Values\Content\Content;
+use eZ\Publish\Core\Repository\Values\Content\ContentCreateStruct;
```

Comment:
One more thing I missed - you should almost always type hint API. See for instance the issue reported by PhpStorm for `createContent` method return value.

In general PhpStorm should not report any issues for the new code (we still have a lot of work to do to enable static analysis for the entire repo).